### ISSUE TITLE
Cannot get very big data (like 100M)

### ISSUE THREAD

**Issue by wanming** [falsecz/hbase-rpc-client#53]
// hbase shell: create 'file', 'content'
Here is the code base on node.js v8.4:
```javascript
const uuid = require('uuid/v4')
const hbase = require('hbase-rpc-client')
const Bluebird = require('bluebird')

const client = hbase({
  zookeeperHosts: ['localhost:2181'],
  zookeeperRoot: '/hbase',
  zookeeperReconnectTimeout: 20000,
  rootRegionZKPath: '/meta-region-server',
  rpcTimeout: 120000,
  callTimeout: 60000,
  tcpNoDelay: true,
  tcpKeepAlive: true
})

Bluebird.promisifyAll(client)

const main = async () => {
  const rowKey = uuid()
  console.log(rowKey)
  console.time('put')
  const put = new hbase.Put(rowKey)
  put.add('content', 'value', Buffer.alloc(1e10, 'a').toString())
  console.log('put...')
  await client.putAsync('file', put)
  console.timeEnd('put')

  console.time('get')
  const get = new hbase.Get(rowKey)
  await client.getAsync('file', get)
  console.timeEnd('get')
}

main().then(console.log).catch(console.log)

```

I got get error 

```
{ Error: timedout
    at maybeWrapAsError (/Users/tom/tom/heq-test/node_modules/bluebird/js/release/util.js:61:12)
    at /Users/tom/tom/heq-test/node_modules/bluebird/js/release/nodeback.js:38:50
    at callback (/Users/tom/tom/heq-test/node_modules/hbase-rpc-client/lib/client.js:453:25)
    at /Users/tom/tom/heq-test/node_modules/hbase-rpc-client/lib/client.js:466:26
    at Call.cb (/Users/tom/tom/heq-test/node_modules/protobufjs/ProtoBuf.js:2963:45)
    at Timeout._onTimeout (/Users/tom/tom/heq-test/node_modules/hbase-rpc-client/lib/call.js:21:24)
    at ontimeout (timers.js:469:11)
    at tryOnTimeout (timers.js:304:5)
    at Timer.listOnTimeout (timers.js:264:5)
  cause:
   Error: timedout
       at maybeWrapAsError (/Users/tom/tom/heq-test/node_modules/bluebird/js/release/util.js:61:12)
       at /Users/tom/tom/heq-test/node_modules/bluebird/js/release/nodeback.js:38:50
       at callback (/Users/tom/tom/heq-test/node_modules/hbase-rpc-client/lib/client.js:453:25)
       at /Users/tom/tom/heq-test/node_modules/hbase-rpc-client/lib/client.js:466:26
       at Call.cb (/Users/tom/tom/heq-test/node_modules/protobufjs/ProtoBuf.js:2963:45)
       at Timeout._onTimeout (/Users/tom/tom/heq-test/node_modules/hbase-rpc-client/lib/call.js:21:24)
       at ontimeout (timers.js:469:11)
       at tryOnTimeout (timers.js:304:5)
       at Timer.listOnTimeout (timers.js:264:5),
  isOperational: true }
```

Any idea ?

**Comment by wision** [comment]
There is incorrect usage of debug module in put (and possibly other) method. I'll have to fix that.

**Comment by wision** [comment]
Fixed in v0.1.11.
